docs/roadmap.md

```diff
@@ -1,5 +1,6 @@
 ## Roadmap
 
+Wiki Link: https://wiki.adorsys.de/display/BG/OBG+Roadmap
```

Comment:
Please no adorsys wiki links on github. Other can not open it...

Reply:
I deleted the wiki link